java-checks/src/test/files/checks/AssertionsInTestsCheck/Junit4.java

```diff
@@ -116,6 +119,14 @@ public void mockito_assertion_verify_no_more_interactions() {
     Mockito.verifyNoMoreInteractions(Mockito.mock(List.class));
   }
 
+  @Test
+  public void selenide_verify_should() {
```

Comment:
This test deserves to be in a separate file dedicated to Selenide framework. See [`AssertionsInTestsCheckTest.java`](https://github.com/SonarSource/sonar-java/blob/master/java-checks/src/test/java/org/sonar/java/checks/AssertionsInTestsCheckTest.java)